bases/renku_data_services/data_api/app.py

```diff
@@ -39,6 +44,25 @@ def str_to_slug(value: str) -> Slug:
         raise ValueError("Couldn't parse slug")
 
 
+def _patched_validate_body(
+    validator: Callable[[type[Any], dict[str, Any]], Any],
+    model: type[Any],
+    body: dict[str, Any],
+) -> Any:
+    """Validate body method for monkey patching.
+
+    sanic_ext does not return contained exceptions as errors anymore, instead it returns a string.
+    This undoes that change.
+    """
+    try:
+        return validator(model, body)
+    except VALIDATION_ERROR as e:
+        raise ValidationError(
+            f"Invalid request body: {model.__name__}. Error: {e}",
+            extra={"exception": e},
+        ) from None
```

Comment:
Out of curiosity: why not `raise ... from e`?

Reply:
this is just the code how it was on their end before they made it even worse. But good point, if we monkepatch it, we might as well handle errors properly